my_game.py

```diff
@@ -46,6 +46,25 @@
 UFO_SHOT_SPEED = 2
 UFO_FIRE_RATE = 1.5
 
+
+def wrap(sprite: arcade.Sprite):
+    """
+    if sprite is off-screen move it to the other side of the screen
+    """
+
+    if sprite.right < 0:
+        sprite.center_x += SCREEN_WIDTH
+
```

Comment:
Please make two else if blocks for x and y respectively.

Reply:
You mean elif?

Reply:
Yes!